### PR TITLE
Show entire step post signup

### PR DESCRIPTION
**Pull request by deadlybutter**
Currently, we modify the store which breaks the action page post signup. This makes a copy of the array so it works on re-render.

### CODE REVIEW

File: resources/assets/components/ActionPage/index.js [hunk 1]
```diff
@@ -56,11 +56,13 @@ const renderStep = (step, index) => {
  * @returns {XML}
  */
 const ActionPage = ({ steps, callToAction, campaignId, signedUp, hasPendingSignup, isAuthenticated, clickedSignUp }) => {
+  let actionSteps = JSON.parse(JSON.stringify(steps));
```
Comment (DFurnes) [inline resources/assets/components/ActionPage/index.js]: 🤔

Comment (DFurnes) [inline resources/assets/components/ActionPage/index.js]: Can you try this with Lodash's [`cloneDeep`](https://lodash.com/docs/#cloneDeep)?

```js
import cloneDeep from 'lodash/cloneDeep';

let actionSteps = cloneDeep(steps);
```

**Comment by deadlybutter** [comment]
@DFurnes Lodash worked!

**Comment by ngjo** [comment]
[🐛 Parts of Step 2 in the Action page are hidden even after Affiliation 🐛](https://trello.com/c/dTMBPsVl/539-parts-of-step-2-in-the-action-page-are-hidden-even-after-affiliation)

**Pull request merged**